WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+ONE_MINUTE_IN_CENTISECONDS = 60*100
```

Comment:
I know I'm an outlier in the centisecond issue, so don't hesitate to ignore this, but...

The SI way to write centisecond is `cs` Would `ONE_MINUTE_IN_cs` make sense? We could also monkey patch in `1.minute.in_cs`.